devito/function.py

```diff
@@ -950,6 +950,11 @@ def __init__(self, *args, **kwargs):
             else:
                 raise TypeError("`space_order` must be int or 3-tuple of ints")
```

Comment:
shouldn't it allow me to do something like:

`g = Grid((10,10))
f = TimeFunction(name='f',grid=g, space_order=(2,2))`

?

Reply:
That section of code has not been modified in this branch.
Does the `TimeFunction` declaration you wrote not work in Master?

Reply:
@btrb no it doesn't. The tuple is` (order, halo_left, halo_right)` as said in the docstring, it's not multiple orders for different  dimensions